package.json

```diff
@@ -60,10 +60,11 @@
     "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.803.25",
-    "@angular/cli": "~8.3.25",
+    "@angular-devkit/build-angular": "0.803.26",
+    "@angular/cli": "~8.3.26",
     "@angular/compiler-cli": "8.1.0",
     "@angular/language-service": "8.1.0",
+    "@babel/compat-data": "~7.8.0",
```

Comment:
You don't need this fix - it's already included in the updated `@angular/cli` and `@angular-devkit/build-angular` :)